models/viralassembly/viralAssemblyModel.m

```diff
@@ -95,37 +95,43 @@ function viralAssemblyModel ()
 
 	D_max = 0.001;   % Maximum number of gapped DNA in nucleus
 
+	MaxP6 = 0.5 * ; % made up parameters
+	k_log = 0.01; % made up parameters
+	p6_activation = 
+
+	RNAiFactor = MaxP6 / (1 + exp(-k_log(P_6 - 0.5)));
```

Comment:
This should be updated:

`MaxP6` is not a descriptive name, it should be max_gamma (since maximum of the curve's y-axis is a degradation rate
add new variable `midP6`, should be half of steady-state P6 with no RNAi (will need to hardcode), replace `P6 - 0.5` with `P6 - midP6`
should not be `-k_log`, replace with just 'k_log` (otherwise curve will not have logistic shape)